lib/screens/more_details_page.dart

```diff
@@ -0,0 +1,223 @@
+    initGetQuestionDetails(id);
+    super.initState();
+  }
+
+  void initGetQuestionDetails(String id) {}
+
+  Column _headerStack() {
+    return Column(
+      children: [
+        Container(
+          height: 0,
+        ),
+        _profileNameAndIcon(),
+        _answerQuestion(),
+      ],
+    );
+  }
+
+  Row _profileNameAndIcon() {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.start,
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        CircleAvatar(
+          radius: 55.0,
+          backgroundColor: Colors.grey[300],
+          child: CircleAvatar(
+            radius: 40.0,
+            backgroundImage: AssetImage('assets/images/dn.jpg'),
+          ),
+        ),
+        Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            Text(
+              "Question Title here",
+              style: TextStyle(fontSize: 25.0, fontWeight: FontWeight.bold),
+              textAlign: TextAlign.left,
+            ),
+            Text(
+              "Technology - Computer Hardware",
+              style: TextStyle(fontSize: 14.0, fontWeight: FontWeight.w300),
+              textAlign: TextAlign.left,
+            ),
+            Text(
+              "Asked : 2 days ago",
+              style: TextStyle(fontSize: 14.0, fontWeight: FontWeight.bold),
+              textAlign: TextAlign.left,
+            ),
+            Padding(
+              padding:
+                  const EdgeInsets.only(left: 0, top: 30, right: 0, bottom: 0),
+              child: RichText(
+                text: TextSpan(
+                    // set the default style for the children TextSpans
+                    style: TextStyle(
+                        fontSize: 14.0,
+                        fontWeight: FontWeight.bold,
+                        color: Colors.black),
+                    children: [
+                      TextSpan(
+                        text: 'Attached media: ',
+                      ),
+                      TextSpan(
+                          text: 'Image1',
+                          style: TextStyle(
+                              color: Colors.blue,
+                              decoration: TextDecoration.underline)),
+                      TextSpan(
+                        text: '  ',
+                      ),
+                      TextSpan(
+                          text: 'Image2',
+                          style: TextStyle(
+                              color: Colors.blue,
+                              decoration: TextDecoration.underline)),
+                    ]),
+              ),
+            ),
+          ],
+        ),
+      ],
+    );
+  }
+
+  Column _answerQuestion() {
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.end,
+      children: [
+        // add if else statement here : if username logged in matches username of question creator -> _questionArchiveButton() else call _answerQuestionButton()
```

Comment:
Please provide the if else. But you can use a fake user id and then compare it with the another mock user id.

Reply:
done